public/locations-list-map.js

```diff
@@ -389,7 +455,7 @@ function initMap(states) {
                  const open_accepted = entry["open_box"];
                  // Convert the lat and lng fields to numbers
                  if (!isNaN(Number(latitude))) {
-                     var marker = addMarkerToMap(map, Number(latitude), Number(longitude),
+                     var marker = addMarkerToMap(Number(latitude), Number(longitude),
```

Comment:
nit: var -> const

Reply:
Yeah, not my code, only changed map. If u want me to reformat entire file I can potentially do it in this pr or a new pr as long as we merge it soon given the momentum of the site and the apparent risk of conflicts.... lmk @awong-dev @graemer 

Lots of then it’s here on code I didn’t write, only edit...

Reply:
skipping; not my code, not the right branch (also for efficiency)